packages/common/src/types/field.ts

```diff
@@ -123,6 +123,14 @@ export type FieldId = string;
 export const fieldId = (field: Pick<Field, 'table' | 'name'>): FieldId =>
     `${field.table}_${field.name}`;
 
+export const convertFieldIdToFieldRef = (field: Field) =>
```

Comment:
This is a duplicate of `getFieldRef` util function. We can delete this one :)

Reply:
ahhh, thanks! removed